cmd/amp/cli/cli.go

```diff
@@ -0,0 +1,36 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"io/ioutil"
+
+	"github.com/fatih/color"
+	"gopkg.in/yaml.v2"
+	"github.com/mitchellh/go-homedir"
+)
+
+// PrintErr prints the error and then exits
+func PrintErr(err error) {
+	color.Set(color.FgRed)
+	fmt.Println(err)
+	os.Exit(1)
+}
+
+// SaveConfiguration saves the configuration to ~/.ampswarm.yaml
+func SaveConfiguration(c interface{}) (err error) {
+	homedir, err := homedir.Dir()
+	if err != nil {
+		return
+	}
+	contents, err := yaml.Marshal(c)
+	if err != nil {
+		return
+	}
+	err = ioutil.WriteFile(path.Join(homedir, ".ampswarm.yaml"), contents, os.ModePerm)
```

Comment:
Please rename `.ampswarm.yaml` -> `.amp.yaml` (corresponds to the executable name)